### PR TITLE
Fix: return error for missing files in index

### PR DESCRIPTION
**Pull request by bobhageman**

### CODE REVIEW

**Review by sanderhollaar — APPROVED** [review]

File: schemes.go [hunk 1]
```diff
@@ -1204,7 +1204,7 @@ func (scheme *SchemeManager) verifyFiles(conf *Configuration) error {
 			return err
 		}
 		if !exists {
-			continue
+			return errors.Errorf("file %s in index is not found on disk", file)
```
Comment (sanderhollaar) [inline schemes.go:1207]: remove `on disk`

Comment (bobhageman) [reply schemes.go:1207]: @sanderhollaar , mentioned the 'on disk' part to keep the error messages consistent with the existing error which can happen when the verifying a requestor-scheme here:

https://github.com/privacybydesign/irmago/blob/fix-scheme-verify/schemes.go#L1376:
```
return errors.Errorf("file %s of requestor scheme %s in index but not found on disk", file, scheme.ID), SchemeManagerStatusParsingError
```

What do you suggest: remove both occurrences?

Comment (sanderhollaar) [reply schemes.go:1207]: hmz I think the `on disk` is meaningless but I understand you want to keep the messages consistent. I'm oke with either solution :-)

Comment (bobhageman) [reply schemes.go:1207]: Then I'll keep it like this

**Pull request merged**